discord/ext/commands/pagination.py

```diff
@@ -0,0 +1,54 @@
+import discord, asyncio
+from typing import List
+from .context import Context
+from discord.ui.button import button
+
+class Paginate(discord.ui.View):
+    """Creates a paginator for embeds that is navigated with buttons."""
+
+    def __init__(self, ctx: Context, embeds: List[discord.Embed]):

```

Comment:
Since `Context` is specific to the commands extension, it might be better off to not use it if we aren't putting this in the commands extension.

I also think we might be able to get away with a `channel` instead?

Reply:
I actually need to change that to use Context or Interaction Context since they are different. But yeah if we move it over, i'll make sure to make a change. 

After giving it some thought though, i cant see the commands extension going away since people with small bots can still make use of it after the big April update.

Reply:
What if I want to use a paginator without having an interaction context?

Would be a shame to limit it.

Reply:
Oh I see what you mean now. I'll work on that.